test/powershell/Modules/Microsoft.PowerShell.Management/Get-Content.Tests.ps1

```diff
@@ -287,26 +287,17 @@ baz
 }
 
 Describe "Get-Content -Raw test" -Tags "CI" {
-    BeforeAll {
-        $lfTextUnterminated = "a`nb"
-        $lfText = "a`nb`n"
-        $crlfTextUnterminated = "a`r`nb"
-        $crlfText = "a`r`nb`r`n"
-        Set-Content -Encoding Ascii -NoNewline "$TestDrive\lf.txt" -value $lfText
-        Set-Content -Encoding Ascii -NoNewline "$TestDrive\crlf.txt" -value $crlfText
-        Set-Content -Encoding Ascii -NoNewline "$TestDrive\lf-nt.txt" -value $lfTextUnterminated
-        Set-Content -Encoding Ascii -NoNewline "$TestDrive\crlf-nt.txt" -value $crlfTextUnterminated
-    }
-    It "Reads LF-terminated files in full." {
-        Get-Content -Raw "$TestDrive\lf.txt" | Should BeExactly $lfText
-    }
-    It "Reads CRLF-terminated files in full." {
-        Get-Content -Raw "$TestDrive\crlf.txt" | Should BeExactly $crlfText
-    }
-    It "Reads LF-separated files without trailing newline in full." {
-        Get-Content -Raw "$TestDrive\lf-nt.txt" | Should BeExactly $lfTextUnterminated
-    }
-    It "Reads CRLF-separated files without trailing newline in full." {
-        Get-Content -Raw "$TestDrive\crlf-nt.txt" | Should BeExactly $crlfTextUnterminated
+
+    $testcases = @(
+                  @{character = "a`nb`n"; testname = "LF-terminated files"; filename = "lf.txt"}
+                  @{character = "a`r`nb`r`n"; testname = "CRLF-terminated files"; filename = "crlf.txt"}
+                  @{character = "a`nb"; testname = "LF-separated files without trailing newline"; filename = "lf-nt.txt"}
+                  @{character = "a`r`nb"; testname = "CRLF-separated files without trailing newline"; filename = "crlf-nt.txt"}
+                )
```

Comment:
Please move to `It` block:
```powershell
It "Reads - <testname> in full" -TestCases @(
    @{character = "a`nb`n"; testname = "LF-terminated files"; filename = "lf.txt"} 
    ....
) {
    ....
}

Reply:
Thanks for your comment. But I would rather not change it as it is just format preference and the submitted format is following the [testing guidelines](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#parameter-generation) example.

Reply:
It is new code not formatting. We should place it in a right place. The right place is either `It` block ([sample](https://github.com/PowerShell/PowerShell/blob/master/test/powershell/Modules/Microsoft.PowerShell.Management/ControlService.Tests.ps1#L12)) or `BeforeAll` block. We use the set only one time so `It` block is best place. This does not contradict the testing guidelines.